.github/workflows/ci.yaml

```diff
@@ -63,8 +63,7 @@ jobs:
       - name: Check Code
         shell: powershell
         run: |
-          # Work around runtime failures on the GH Actions runner
```

Comment:
Can we please keep this comment (the nuget add source is still working around runtime failures after all), and squash the commits together?